ci-operator/step-registry/windows/ci/fastforward/windows-ci-fastforward-commands.sh

```diff
@@ -34,7 +34,7 @@ if [[ -z "$SOURCE_BRANCH" ]]; then
 fi
 
 if [[ -z "$DESTINATION_BRANCH" ]]; then
-    echo "ERROR DESTINATION_BRANCH may not be empty"
+    echo "ERROR $DESTINATION_BRANCH may not be empty"
```

Comment:
Since this is an error message like the previous lines, the variable does not need to be expanded here.  The `$` can be dropped.